.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,91 @@
+name: Bug report
+description: Are you getting false positives? Is a supported rule not working for you? Open a bug report.
+title: 'Bug report'
+body:
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: Here you can summarise what is going on.
+      placeholder: e.g. ESLint marked an unused variable starting with `_` as unused, even though I added this pattern to my `.eslintrc`.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: Describe what you would expect to happen instead.
+      placeholder: e.g. I expected my linter to pass.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Reproduction steps
+      description: Providing a set of concise steps that deterministically reproduce the behaviour helps us investigate what is going on.
+      placeholder: |
+        For example:
+        1. I extended the `@code-pushup/eslint` in my `.eslintrc`.
+        2. I added explicit options to the `no-unused-vars` rule.
+        3. When I opened the `index.ts` file, ESLint highlighted the `no-unused-vars` rule despite the variable `_unused` matching the pattern I explicitly set.
+    validations:
+      required: true
+  - type: input
+    id: working-version
+    attributes:
+      label: Has this worked for you before?
+      description: Provide the last working version.
+      placeholder: e.g. 0.5.0
+  - type: dropdown
+    id: config
+    attributes:
+      label: Which configuration is this related to?
+      options:
+        - default
+        - TypeScript
+        - Node.js
+        - Angular
+        - NgRx
+        - Jest
+        - Vitest
+        - Cypress
+        - Storybook
+    validations:
+      required: true
+  - type: textarea
+    id: plugin-versions
+    attributes:
+      label: Plugin and dependencies version
+      description: What plugin version are you running? Include peer dependencies if applicable.
+      placeholder: |
+        For example:
+        `@code-pushup/eslint-config/typescript` version 0.5.0
+
+        Peer dependencies:
+        - @typescript-eslint/eslint-plugin version 7.4.0
+    validations:
+      required: true
+  - type: input
+    id: eslint-version
+    attributes:
+      label: ESLint version
+      description: What ESLint version are you running? (check in your `package.json`)
+      placeholder: e.g. 8.57.0
+    validations:
+      required: false
+  - type: input
+    id: ts-version
+    attributes:
+      label: TypeScript version
+      description: What TypeScript version are you running? (check with `tsc -v` or in `tsconfig.base.json`)
```

The version isn't mentioned in `tsconfig.json`.

```suggestion
      description: What TypeScript version are you running? (check with `npx tsc -v`)
```

---

You are right, it only references ES version.